### DISCUSSION TITLE
No snapshots in captures folder

### DISCUSSION THREAD

**Discussion by SilverRain3** [djdd87/SynoAI#167] [Q&A]
I've managed to set this up and thank you!
No more unwanted notification here!!

Pushbullet notification are working.
However I'm missing captures in the folder on my NAS. What am I missing/doing wrong?

JSON file:

{
  "Logging": {
    "LogLevel": {
      "Default": "Warning",
      "Microsoft": "Warning",
      "Microsoft.Hosting.Lifetime": "Warning"
    }
  },

  "Url": "http://192.168.1.50:5000",
  "User": "xxxblablaxxx",
  "Password": "xxxblablaxxx",

  "MinSizeX": 100,
  "MinSizeY": 100,
  "DaysToKeepCaptures": 14,
  
  "AI": {
    "Type": "DeepStack",
    "Url": "http://192.168.1.50:83"
  },

  "Notifiers": [
    {
      "Type": "Pushbullet",
      "ApiKey": "xxxblablaxxx"
    },

  ],

  "Cameras": [
    {
      "Name": "Straat",
      "Types": [ "Person", "Car" ],
      "Threshold": 70
    },
    {
      "Name": "Tuin",
      "Types": [ "Person" ],
      "Threshold": 45
    },
    {
      "Name": "Veranda",
      "Types": [ "Person" ],
      "Threshold": 45
    },
    {
      "Name": "Poort",
      "Types": [ "Person" ],
      "Threshold": 45
    }
   ]
}

**Comment by djdd87** [comment]
What have you setup for your volume mappings?

**Comment by SilverRain3** [comment]
/volume1/docker/synoai/captures ==> /app/captures

/volume1/docker/synoai/appsettings.json ==> /app/appsettings.json

**Reply by djdd87** [reply]
I can't remember if it's case sensitive or not. Can you put a capital C on Captures on the target path, as per the docs, and try again?

**Reply by SilverRain3** [reply]
That was the solution!
Thank you!!
